musica/micm/micm.F90

```diff
@@ -1,201 +1,200 @@
 module micm
-   ! Wrapper for MICM functionality
-   use iso_c_binding
+  use iso_c_binding
 
-   ! Note: "micm_core" is included in an external pre-built MICM library that the host
-   ! model is responsible for linking to during compilation
-   use micm_core, only: micm_t
-   use ccpp_kinds, only: kind_phys
+  ! Note: "micm_core" is included in an external pre-built MICM library that the host
+  ! model is responsible for linking to during compilation
+  use micm_core, only: micm_t
+  use ccpp_kinds, only: kind_phys
 
-   implicit none
+  implicit none
 
-   public :: micm_init, micm_run, micm_final
-   private :: convert_to_mol_per_cubic_meter, convert_to_mass_mixing_ratio
+  public :: micm_init, micm_run, micm_final
+  private :: convert_to_mol_per_cubic_meter, convert_to_mass_mixing_ratio
 
-   type(micm_t), allocatable :: micm_obj
+  type(micm_t), allocatable :: micm_obj
 
 contains
 
-   !> \section arg_table_micm_init Argument Table
-   !! \htmlinclude micm_init.html
-   subroutine micm_init(config_path, iulog, errcode, errmsg)
-       ! Initializes MICM objects by creating solvers from configure files
-
-       character(len=*), intent(in)     :: config_path
-       integer, intent(in)              :: iulog
-       integer, intent(out)             :: errcode
-       character(len=512), intent(out)  :: errmsg
- 
-       errcode = 0
-       errmsg = ''
- 
-       ! Constructs MICM object
-       allocate(micm_obj)
-       micm_obj = micm_t(config_path)
- 
-       ! Creates solver
-       errcode = micm_obj%create_solver()
- 
-       if (errcode /= 0) then
-           !TODO(jiwon) - which file?
-           errmsg = "INIT MICM: FATAL: Failed in creating MICM solver because parsing &
-                   configuration files failed. Please look over at the other file."  
-       else 
-           write(iulog,*) "   [INIT MICM]: INFO: Created MICM solver"
-       endif
+  !> \section arg_table_micm_init Argument Table
+  !! \htmlinclude micm_init.html
+  subroutine micm_init(config_path, iulog, errcode, errmsg)
+    character(len=*), intent(in)    :: config_path
+    integer, intent(in)             :: iulog
+    integer, intent(out)            :: errcode
+    character(len=512), intent(out) :: errmsg
+
+    errcode = 0
+    errmsg = ''
+
+    allocate(micm_obj)
+    micm_obj = micm_t(config_path)
+
+    errcode = micm_obj%create_solver()
+
+    if (errcode /= 0) then
+      errmsg = "[fatal] [micm] Failed to creat MICM solver. Parsing configuration failed. &
+                Please look over at the other file for further information."
```

Comment:
I think there is a typo here (`creat` -> `create`).  Also would it be better to specifically say something like `MICM log file` instead of `other file`, or do we not know what the other file actually is yet?

Reply:
Thank you for catching the typo! I missed it. Yeah I was also wondering where the log messages are to be written. Is it possible to create `micm` only log file or would they be part of the other file, ie. atmospheric physics log?

Reply:
It should be possible to have `micm` create it's own log file, or have that log information included in the CAM-SIMA log file (or both).  Just let me know if you or the rest of the MUSICA team have a preference.